From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 1 Nov 2018 13:40:37 +0100
Subject: [PATCH] media: coda: set V4L2_CAP_TIMEPERFRAME flag in coda_s_parm

The flag is already set in coda_g_parm, but v4l2-compliance complains
about it not being set during S_PARM.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
---
 drivers/media/platform/coda/coda-common.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/platform/coda/coda-common.c b/drivers/media/platform/coda/coda-common.c
index 6dbf06eb9521..f44cf7f9e9b7 100644
--- a/drivers/media/platform/coda/coda-common.c
+++ b/drivers/media/platform/coda/coda-common.c
@@ -1242,9 +1242,11 @@ static int coda_s_parm(struct file *file, void *fh, struct v4l2_streamparm *a)
 
 	switch (a->type) {
 	case V4L2_BUF_TYPE_VIDEO_OUTPUT:
+		a->parm.output.capability = V4L2_CAP_TIMEPERFRAME;
 		tpf = &a->parm.output.timeperframe;
 		break;
 	case V4L2_BUF_TYPE_VIDEO_CAPTURE:
+		a->parm.capture.capability = V4L2_CAP_TIMEPERFRAME;
 		tpf = &a->parm.capture.timeperframe;
 		break;
 	default:
